src/pages/ManageComputeHost/DeactivateComputeHost.js

```diff
@@ -55,17 +48,17 @@ class DeactivateComputeHost extends DisableComputeServiceNetwork {
   }
 
   getPlaybooks() {
-    let playbooks = [];
-    if (this.canMigrate()) {
-      playbooks.push({
-        name: MIGRATE_INSTANCES,
-        action: ::this.migrateInstances
-      });
+    let playbooks = [{
+      name: constants.DISABLE_COMPUTE_SERVICE_ACTION,
+      action: ::this.disableCompServices
+    }];
+
+    let instancesPlaybooks = this.getPlaybooksForInstances();
```

Comment:
Ditto above, these 4 lines should just be `playbooks = playbooks.concat(this.getPlaybooksForInstances());`